packages/jose/jose.0.1.0/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+synopsis: "JOSE implementation for OCaml and ReasonML"
+description:
+  "JavaScript Object Signing and Encryption built ontop of pure OCaml libs"
+maintainer: ["ulrik.strid@outlook.com"]
+authors: ["Ulrik Strid"]
+license: "MIT"
+homepage: "https://ulrikstrid.github.io/reason-jose"
+doc: "https://ulrikstrid.github.io/reason-jose"
+bug-reports: "https://github.com/ulrikstrid/reason-jose/issues"
+depends: [
+  "ocaml" {>= "4.06.0"}
+  "base64"
+  "dune" {>= "1.11" & < "2.0"}
```

Comment:
I'm intrigued, what kind of problem did you encountered to not be compatible with dune 2.0 if you can elaborate on that?

Reply:
Nocrypto's dune branch wasn't up-to-date and used `dune.configuration` which doesn't exist in 2.0. This is a bandaid for that basically, but this should be changed when nocrypto has a proper release.